Tenduke.Client.DefaultBrowser/Authorization/AuthorizationCodeGrant.cs

```diff
@@ -19,9 +19,7 @@ namespace Tenduke.Client.DefaultBrowser.Authorization
     /// <summary>
     /// Authorization Code Grant flow using OS default browser.
     /// </summary>
-    [Serializable]
```

Comment:
`[Serializable]` is for Binary or XML serialization - the code is now using JSON serialization directly so the attribute is not needed anymore.